lib/jsdom/living/custom-elements/CustomElementRegistry-impl.js

```diff
@@ -185,6 +185,7 @@ class CustomElementRegistryImpl {
       localName,
       constructor,
       objectReference: ctor,
+      formAssociated: ctor.formAssociated,
```

Comment:
Implement this more closely following the spec in https://html.spec.whatwg.org/#element-definition . Steps 10, and 14.11-14.12. It doesn't make a practical difference right now because formAssociated is the last property gotten, but the order would be observable if the spec added another property getting step in the future.